### PR TITLE
ODC-7666: Used LogViewer of PF for pipeline logs

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Story:**
https://issues.redhat.com/browse/ODC-7666

**Description:**
LogViewer component of PatternFly is used for Pipeline logs

**Demo:**
https://drive.google.com/file/d/1dPHC8a7gMS7azkB0ExSKtSeh_609nZFZ/view?usp=drive_link
https://drive.google.com/file/d/1Hf6UrFVLqh45ZfqV4hbZ11yOt12NHSGk/view?usp=drive_link

### CODE REVIEW

**Comment by lokanandaprabhu** [comment]
Created issues in https://github.com/patternfly/react-log-viewer/issues

https://github.com/patternfly/react-log-viewer/issues/78 - While scrolling in LogViewer, if the log content consists of ANSI color code, then after, on scroll up or scroll down, different colors will show up on the connect

https://github.com/patternfly/react-log-viewer/issues/77 - LogViewer is not maintaining indentation/removing spacing which is prefix to the log data

File: src/components/logs/TektonTaskRunLog.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+import { LogViewer } from '@patternfly/react-log-viewer';
 import { HttpError } from '@openshift-console/dynamic-plugin-sdk/lib/utils/error/http-error';
 import { TaskRunKind } from '../../types';
 import { TektonResourceLabel } from '../../consts';
-import './Logs.scss';
-import './MultiStreamLogs.scss';
 import { LoadingInline } from '../Loading';
 import { useTRTaskRunLog } from '../hooks/useTektonResult';
+import './Logs.scss';
+import './MultiStreamLogs.scss';
```
Comment (vikram-raj) [inline src/components/logs/TektonTaskRunLog.tsx:9]: Can we have all CSS for logs in one file now?

Comment (lokanandaprabhu) [reply src/components/logs/TektonTaskRunLog.tsx]: Updated the PR. PTAL.

**Review by vikram-raj — COMMENTED** [review]
Verified looks fine. 

Couple of issues in the LogViewer component that need to be followup and fix. Which has been reported https://github.com/patternfly/react-log-viewer/issues/78 and https://github.com/patternfly/react-log-viewer/issues/77

Merging this PR and will create a followup PR once issues get fixed

/lgtm

**Pull request merged**